api/controllers/UserController.js

```diff
@@ -64,7 +64,7 @@ module.exports = {
       // Query for game if user is in one
       const gameId = (user.game ?? req.session.game) ?? null;
 
-      if (process.env.VITE_USE_GAMESTATE_API) {
+      if (process.env.VITE_USE_GAMESTATE_API && gameId) {
```

Comment:
This block is logic for reproducing the latest socket event for a specific game, so it's only needed when the there is a game in question. It was erroring when we hit `reLogin` in the gamestate api without a gameId provided.